dspace/config/registries/relationship-formats.xml

```diff
@@ -9,11 +9,116 @@
         <namespace>http://dspace.org/relationship</namespace>
     </dc-schema>
 
+    <dc-schema>
+        <name>relation</name>
+        <namespace>http://dspace.org/relation</namespace>
+    </dc-schema>
 
     <dc-type>
         <schema>relationship</schema>
         <element>type</element>
         <scope_note>Metadata field used for the type of entity, stored in the item</scope_note>
```

Comment:
@benbosman : This is a question more than a request.  This PR is pointing out the oddity of the "relationship.type" metadata field. Why is this field called "relationship.type"?  Shouldn't it be something like "item.type" (as it's the type of Item, and has nothing to do with the type of *relationship*).  Have a statement like "relationship.type = Author" seems very odd/confusing to me.

This PR is making that odd name evident as you now are creating a separate "relation" schema which includes actual *types* of relationships (isAuthorOfPublication, isPublicationOfAuthor, etc)